app/reducers/elements.js

```diff
@@ -1,10 +1,19 @@
 
 const initialState = {
   textBox: {
+    /*
     1: { top: 20, left: 80, size: 'large', text: 'My vacay memories', background: '#51b6ea', color: '#000000' },
```

Comment:
maybe we should just leave one of these commented out in each type and label it example. And replace the key value with 'uniqueId'